tests/test_hf_file_system.py

```diff
@@ -310,14 +340,14 @@ def test_find_root_directory_no_revision_with_incomplete_cache(self):
             repo_type="dataset",
         )
 
-        files = self.hffs.find(self.hf_path, detail=True)
+        files = copy.deepcopy(self.hffs.find(self.hf_path, detail=True))
```

Comment:
Why do we need a deep copy? Is it because `files` is a reference to an internal object of `self.hffs.find`? If that's the case it would be a bit worrying :confused:

Reply:
Maybe I can fix `find` to return a copy of the cache instead of the cache content itself (that users could modify)